modules/ROOT/pages/tgw-manage-arm.adoc

```diff
@@ -0,0 +1,101 @@
+= Manage Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+
+Use Runtime Manager to manage transit gateway attachments.
```

Comment:
I recommend using the full spelling at first use in a Level-1 topic:
```suggestion
Use Anypoint Runtime Manager to manage transit gateway attachments.
```

Reply:
Anything else to say about the "why" of the task?
Maybe add something that specifies some or all of the subtopics?